app/views/discussions/terms.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="mu-user-header">
+  <h1><%= t :forum_terms %></h1>
+</div>
+
+<div>
+  <% @forum_terms.each do |term| %>
+    <%= term.content_html %>
```

Comment:
I'd recommend adding a semantic class to this parent div, and a div with another semantic class for each term within the foreach. That way, we can easily customize UI using custom css

Reply:
:+1: I missed that in the hurry